bsp/resources/messages/ScalaBspBundle.properties

```diff
@@ -74,13 +74,16 @@ bsp.test.no.class.matches.with.the.regex=No class matches with the regex
 bsp.task.fetching.sources=Fetching source list from BSP
 bsp.task.fetching.jvm.test.environment=Fetching JVM test environment from BSP
 bsp.task.error.could.not.extract.path=Could not extract project path from module {0}
+bsp.task.error.no.class.extractor="Could not find configuration extender for '{0}"
```

Comment:
quotes look wrong

Reply:
Thanks, I also noticed that the `{0}` tokens cannot be surrounded with `'`